backend/app/controllers/spree/admin/payments_controller.rb

```diff
@@ -17,6 +17,8 @@ def index
       end
 
       def new
+        # Move order to payment state in order to capture tax generated on shipments
+        @order.next if @order.delivery?
```

Comment:
@nimish13 What delivery state is skipped in the checkout_flow of order. This would raise an exception?